src/app/pages/article-view/article-view.component.ts

```diff
@@ -62,6 +75,33 @@ export class ArticleViewComponent implements OnInit {
 		window.open(url, '_blank');
 	}
 
+	/**
+	 * @author Marcel Alex
+	 * @summary Changes the Like status of the article
+	 * @description Checks if user is logged in and if so, changes the Like status of the article
+	 * @param {MouseEvent} event - The mouse event to stop propagation
+	 */
+
+	public changeLikeStatus(event: MouseEvent): void {
+		event.stopPropagation();
```

Comment:
warum ist event.stopPropagation() notwendig? :) darf gerne drin bleiben wenns begründet is

Reply:
@jeppard

Reply:
event.stopPropergation is only in the article component needed, because the container there has also a click action. This method prevents the propergation of the click event to parent container. So the article is not shown. In this place it should not be needed